# Spatial Transcriptomics

## Textbook-like documentation

### Single-cell best practices (Chapters 26-29, with 30 & 31 optional)[1]

## Tutorials

### Xenium[2]

### Visium[3]

### Spatial statistics[4]

## Advanced

### Sub-cellular analysis[5]

### Immunofluorescence[6]

### Ligand-Receptor/Cell-Cell Communication

#### Basic[7]

#### Advanced[8][9]

## Recommendations

### Learn the basics of the Xenium, Visium, and Visium HD instruments (some are at single cell, subcellular, and multi-cellular resolution and some have limited gene panels versus whole transcriptome)

### Read the Pang et al. Histopathology article with a closer read of the “Potential uses of spatial transcriptomics in anatomical pathology” section.

### Read through the SC Best Practices chapters, trying out running the targeted tutorials, and (optionally) reading through and/or running any advanced tutorials of interest. It may be worth just glancing at the optional ones, even if you don’t want to read through them closely, just to be aware of what’s out there in terms of analytic possibilities.

[1]: https://www.sc-best-practices.org/spatial/introduction.html
[2]: https://squidpy.readthedocs.io/en/stable/notebooks/tutorials/tutorial_xenium.html
[3]: https://squidpy.readthedocs.io/en/stable/notebooks/tutorials/tutorial_visium_hne.html
[4]: https://gustaveroussy.github.io/sopa/tutorials/spatial/
[5]: https://bento-tools.readthedocs.io/en/latest/tutorial_gallery/Main_Guide.html
[6]: https://squidpy.readthedocs.io/en/stable/notebooks/tutorials/tutorial_visium_fluo.html
[7]: https://dev-st.readthedocs.io/en/latest/tutorials/stLearn-CCI.html
[8]: https://liana-py.readthedocs.io/en/latest/notebooks/bivariate.html
[9]: https://liana-py.readthedocs.io/en/latest/notebooks/misty.html
